MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -32,6 +34,13 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+		viewModel?.onFeedLoadError = { [weak self] message in
+			guard let message = message else {
+				self?.errorView?.hideMessage()
+				return
+			}
+			self?.errorView?.show(message: message)
```

Comment:
`if/else` is more idiomatic and easier to read in this context.

For example: "If there's an error message, show it. Else, hide any existing message."

Instead of: "guard against a `nil` error message. Else, hide the current message and return. Show the error message."
```suggestion
			if let message = viewModel.message {
				self?.errorView?.show(message: message)
			} else {
				self?.errorView?.hideMessage()
			}
```